cmd/boulder-va/main.go

```diff
@@ -80,6 +81,17 @@ func main() {
 		err = rpc.NewValidationAuthorityServer(vas, vai)
 		cmd.FailOnError(err, "Unable to setup VA RPC server")
 
+		if c.VA.GRPC != nil {
+			s, l, err := bgrpc.NewServer(c.VA.GRPC)
+			cmd.FailOnError(err, "Unable to setup VA gRPC server")
+			err = wrappers.RegisterValidationAuthorityGRPCServer(s, vai)
+			cmd.FailOnError(err, "Unable to register VA gRPC server")
+			go func() {
```

Comment:
I agree in principle, but this will make deployment more complex. We would need to generate two copies of the config that are the same in everything except the debugport and the way they export their services (AMQP vs gRPC).